interface/forms/newpatient/templates/newpatient/partials/common/_body-scripts.html.twig

```diff
@@ -0,0 +1,27 @@
+{# Module Theme writers can override this file to add their own fields #}
+<script>
+    (function() {
+        window.document.addEventListener("DOMContentLoaded", function() {
+            if (window.NewPatientForm) {
+                let config = {
+                    webroot: {{ webroot|xlj }},
+                    pid: {{ pid|xlj }},
+                    viewmode: {{ viewmode|xlj }},
+                    encounter: {{ encounter|xlj }},
+                    validationConstraints: {{ validationConstraints|json_encode }},
+                    isPosEnabled: {{ isPosEnabled ? "true" : "false" }},
+                    csrfToken: {{CSRF_TOKEN_FORM|xlj}},
+                    enableGroupTherapy: {{ enableGroupTherapy ? "true" : "false" }},
+                    therapyGroupCategories: {{ therapyGroupCategories|json_encode }},
+                    duplicate: {{ duplicate|json_encode }},
```

Comment:
instead of xlj (note this will translate) and json_encode above, use js_escape

Reply:
Fixed